README.md

```diff
@@ -69,6 +69,18 @@ pnpm run storybook
 
 Storybook will be available at [http://localhost:26638/](http://localhost:26638/)
 
+### Stop Processes
+
+```bash
+npx dev-pm shutdown
+```
+
+### Handy Alias
+
+```
+alias dpm="npm exec -- dev-pm"
+```
+
```

Comment:
the alias is already documented here: https://github.com/vivid-planet/dev-process-manager, so I would remove this